### PR TITLE
Add mnx metabolite search

### PR DESCRIPTION
**Pull request by annasirunian**

### CODE REVIEW

File: src/views/InteractiveMap/ReactionDialog.vue [hunk 1]
```diff
@@ -102,77 +67,63 @@
...
-                  ><v-btn icon @click="addProduct">
+                  ><v-btn icon @click="addSubstrate">
                     <v-icon color="primary">add_circle</v-icon></v-btn
                   >
                   <v-btn
                     icon
-                    v-if="products.length > 1"
-                    @click="deleteProduct(index)"
+                    v-if="substrates.length > 1"
+                    @click="deleteSubstrate(index)"
```
Comment (lukaw3d) [inline src/views/InteractiveMap/ReactionDialog.vue]: Three mistake replacements here

File: src/views/InteractiveMap/ReactionDialogMetabolite.vue [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+import Vue from "vue";
+import MetaboliteDialog from "@/views/InteractiveMap/MetaboliteDialog.vue";
+import * as settings from "@/utils/settings";
+import axios from "axios";
+export default Vue.extend({
+  name: "ReactionDialog",
+  components: {
+    MetaboliteDialog
+  },
+  props: ["model", "customMetabolites", "metabolite", "index"],
```
Comment (lukaw3d) [inline src/views/InteractiveMap/ReactionDialogMetabolite.vue]: "index" isn't used anymore

File: src/views/InteractiveMap/ReactionDialog.vue (hunk 1 above)
Comment (annasirunian) [reply src/views/InteractiveMap/ReactionDialog.vue]: Thanks, fixed

File: src/views/InteractiveMap/ReactionDialog.vue [hunk 3]
```diff
@@ -191,12 +191,12 @@ export default Vue.extend({
...
       if (this.lowerBound >= 0) {
-        return " -> ";
+        return " 🠖 ";
       }
       if (this.upperBound <= 0) {
-        return " <- ";
+        return " 🠔 ";
       }
-      return " ⇌ ";
+      return " ⇄ ";
```
Comment (lukaw3d) [inline src/views/InteractiveMap/ReactionDialog.vue]: Im not sure if this requires a change, but this sign is a bit different
than that returned by bigg api (encoded as `&#8652;` &#8652;)
https://api.dd-decaf.eu/bigg/models/universal/reactions/HPT_h

On the other hand metanetx api returns simple =

File: src/views/InteractiveMap/ReactionDialog.vue [hunk 4]
```diff
@@ -191,12 +191,12 @@ export default Vue.extend({
     },
     direction() {
       if (this.lowerBound >= 0) {
-        return " -> ";
+        return " 🠖 ";
       }
       if (this.upperBound <= 0) {
-        return " <- ";
+        return " 🠔 ";
```
Comment (lukaw3d) [inline src/views/InteractiveMap/ReactionDialog.vue]: These one-way arrows aren't displayed on Windows8.1 (they come from unicode update in 2014)
I would choose something within U+FFFF range
https://www.compart.com/en/unicode/search?q=leftwards+arrow#characters

**Pull request merged**